openslides/core/urls.py

```diff
@@ -12,11 +12,7 @@
         views.VersionView.as_view(),
         name='core_version'),
 
-    url(r'^history/data/$',
-        views.HistoryDataView.as_view(),
-        name='core_history_data'),
-
-    url(r'^history/information/$',
-        views.HistoryInformationView.as_view(),
-        name='core_history_information'),
+    url(r'^history/$',
+        views.HistoryView.as_view(),
```

Comment:
cannot test history anymore.
On  a get requiest to `apps/core/history/`
I get: object has no attribute 'pk'

Even on a completely fresh and migrated database.

Reply:
Should be fixed now.

Reply:
Use query parameter to get filtered data:

apps/core/history/?timestamp=123456789